src/app/wallet/wallet.component.ts

```diff
@@ -1,12 +1,13 @@
 import { Component, OnInit } from '@angular/core';
 import { WalletService } from './wallet.service';
-import { WalletModel } from '../models/WalletModel';
+import { Transaction, TransactionList, WalletModel } from '../models/WalletModel';
 import { SpinnerUtil } from '../utilities/spinner-utilities';
 import { displayBackendError } from '../utilities/error-handler';
 import { centsToBaseCurrencyUnit, prettyCurrency } from '../utilities/currency-util';
 import * as numeral from 'numeral';
 import { ArkaneConnect, SecretType } from '@arkane-network/arkane-connect';
 
+
```

Comment:
I'm not that picky, but try to remove more than 1 extra space in the code. Use one extra space to logically and visually organise your code.